js/src/forum/components/SubscriptionMenu.js

```diff
@@ -59,41 +61,40 @@ export default class SubscriptionMenu extends Dropdown {
       ? 'flarum-subscriptions.forum.sub_controls.notify_email_tooltip'
       : 'flarum-subscriptions.forum.sub_controls.notify_alert_tooltip'));
 
-    const buttonProps = {
+    const buttonAttrs = {
       className: 'Button SubscriptionMenu-button ' + buttonClass,
       icon: buttonIcon,
-      children: buttonLabel,
       onclick: this.saveSubscription.bind(this, discussion, ['follow', 'ignore'].indexOf(subscription) !== -1 ? false : 'follow'),
       title: title
     };
 
     if ((notifyEmail || notifyAlert) && subscription === false) {
-      buttonProps.config = element => {
-        $(element).tooltip({
+      buttonAttrs.onupdate = vnode => {
```

Comment:
Should we set oncreate to the same?

Reply:
What happens right now? Is the tooltip available at all?

Reply:
Ping. It does indeed look like this should happen both for `onupdate` and `oncreate`.